scss/login.scss

```diff
@@ -0,0 +1,66 @@
+.login-container {
+  background: $wild-sand;
+  border-top: 8px solid $cerulean;
+  font-family: 'Roboto', sans-serif;
+  max-width: 300px;
+  padding: 24px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Why you use position absolute and transform to center the login? You can use flex to center an element on the screen.

Reply:
I suggested pablo to do this instead of using a div just to center his login form. He could do that by targeting the body instead, but its not that common to do that or seen as a good practice.

I think that for this case this it's a good approach. The less tags we write in our HTML the better